.github/workflows/connector-publish.yml

```diff
@@ -0,0 +1,97 @@
+name: Connector Publish workflow
+
+permissions: read-all
+
+on:
+  workflow_call:
+    inputs:
+      package-name:
+        required: true
+        type: string
+      branch:
+        required: true
+        type: string
+        default: "main"
+      rust-profile:
+        type: string
+        default: "release"
+      cloud-url:
+        required: true  
+        type: string
+    secrets:
+      CLOUD_USER_EMAIL:
+        required: true
+      CLOUD_USER_PASSWORD:
+        required: true
+
+jobs:
+  linux:
+    name: linux
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        rust-target: [aarch64-unknown-linux-musl, aarch64-unknown-linux-gnu, x86_64-unknown-linux-musl, x86_64-unknown-linux-gnu]  
+    env:
+      CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
+    steps:
+      - name: Install Fluvio
+        run: |
+          curl -fsS https://packages.fluvio.io/v1/install.sh | bash
+          echo "$HOME/.fluvio/bin" >> $GITHUB_PATH
+      - name: Install Fluvio CDK
+        run: fluvio install cdk --develop
+      - name: Fluvio Login
+        run: fluvio cloud login --email ${{ secrets.CLOUD_USER_EMAIL }} --password ${{ secrets.CLOUD_USER_PASSWORD }} --remote ${{ inputs.cloud-url }}
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch }}
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-targets: "false"
+          cache-on-failure: "true"
+      - name: Install Cross
+        run: cargo install cross
+      - name: Build
+        run: |
+          cross build --profile ${{ inputs.rust-profile }}  --target ${{ matrix.rust-target }} -p ${{ inputs.package-name }} 
+          cp target/${{ matrix.rust-target }}/${{ inputs.rust-profile }}/* target/${{ inputs.rust-profile }}/ | true
+      - name: Publish
+        run: |
+          cdk publish --public-yes --target ${{ matrix.rust-target }} -p ${{ inputs.package-name }} 
+
+  macos:
+    name: macos
+    runs-on: macos-latest
```

Comment:
these seems to be same as previous job, why not add as another strategy config?

Reply:
Not exactly the same: 1. runners and targets do not intersect, 2. cargo instead of cross, 3. need to explicitly `rustup target add`.

Reply:
fair enough